### PR TITLE
OnePassMeanVarStd and PPCA examples

### PR DESCRIPTION
**Pull request by ordabayevy**

### CODE REVIEW

File: examples/onepass_mean_var_std.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+"""
+Example: One Pass Mean Var Std
```
Comment (mbabadi) [inline examples/onepass_mean_var_std.py:2]: `Example: One-pass calculation of feature mean, variance, and standard deviation`

File: examples/onepass_mean_var_std.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+==============================
+
+This example shows how to calculate mean, variance, and standard deviation of log normalized
+gene expression count data in one pass [1].
```
Comment (mbabadi) [inline examples/onepass_mean_var_std.py:6]: gene expression count -> feature count

File: examples/onepass_mean_var_std.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    DistributedAnnDataCollection,
+    IterableDistributedAnnDataCollectionDataset,
+    collate_fn,
+)
+from scvid.module.onepass_mean_var_std import OnePassMeanVarStd
+from scvid.train.training_plan import DummyTrainingPlan
+from scvid.transforms import ZScoreLog1pNormalize
+
+
+def main(args):
```
Comment (mbabadi) [inline examples/onepass_mean_var_std.py:33]: You are welcome: https://click.palletsprojects.com/en/8.1.x/

File: examples/probabilistic_pca.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
+"""
+Example: Probabilistic PCA
+==========================
+
+This example shows how to fit gene expression count data to probabilistic PCA
```
Comment (mbabadi) [inline examples/probabilistic_pca.py:5]: feature count :D

**Review by mbabadi — CHANGES_REQUESTED** [review]
Beautiful!

File: examples/onepass_mean_var_std.py (hunk 3 above)
Comment (ordabayevy) [reply examples/onepass_mean_var_std.py:33]: Click is an awesome library! I've used Typer before which is based on Click. Would be nice to incorporate one of these in the future.

**Review by ordabayevy — COMMENTED** [review]
Thanks for reviewing Mehrtash! Addressed your comments.

**Pull request merged**